lib/install.js

```diff
@@ -774,6 +774,9 @@ Installer.prototype.printInstalledForHuman = function (diffs, cb) {
   var added = 0
   var updated = 0
   var moved = 0
+  // Count the number of contributors to packages added, tracking
+  // contributors we've seen, so we can produce a running unique count.
+  var contributorsSet = []
```

Comment:
You can use a `Set` here and get uniqueness for free

Reply:
No compat. issue? Then will do.

Reply:
Yup, we're Node 4+, which includes Set and Map and Symbol!